api/v1beta1/workspace_validation_test.go

```diff
@@ -1252,8 +1252,8 @@ func TestTuningSpecValidateCreate(t *testing.T) {
 		{
 			name: "Invalid Output Destination Casing",
 			tuningSpec: &TuningSpec{
-				Input:  &DataSource{Name: "valid-input", Image: "AZURE_ACR.azurecr.io/input:0.0.0"},
-				Output: &DataDestination{Image: "AZURE_ACR.azurecr.io/OUTPUT:0.0.0", ImagePushSecret: "secret"},
+				Input:  &DataSource{Name: "valid-input", Image: "kaito.azurecr.io/input:0.0.0"},
```

Comment:
why rename

Reply:
`AZURE_ACR.azurecr.io` is not a valid domain. You cannot have uppercase characters or underscores in a domain name. With the new validation method for image names, this was causing errors.